### PR TITLE
Add propValue analysis (#17)

### PR DESCRIPTION
**Pull request by wavebeem**
Continuation of #17 from @Vadimkin

### CODE REVIEW

**Comment by wavebeem** [comment]
Thanks for your hard work on the previous PR @Vadimkin!

I updated the type for `PropValue` and fixed the tests to match. Would you mind trying out this branch in your project? Let me know if it works well.

If you want to try it yourself, you can do this:

1. Clone this repo
2. Switch to the branch `propValue-part-2`
3. `npm pack`
4. `cd` to your other project
5. `npm install {path-to-jsx-info}.tgz`

**Comment by Vadimkin** [comment]
@wavebeem yeah, it works, thanks! 👏

Also, great idea with new `PropValue` type! It's easier to filter needed values by type

File: src/api.ts [hunk 1]
```diff
@@ -42,9 +42,26 @@ export interface ErrorInfo {
...
 }
 
+/**
+ * All possible types of prop values.
+ *
+ * symbol is when prop value can't be represented (expression is used as prop value)
+ * undefined is used when when prop value can't be calculated
+ * null when prop is with {null} value
+ * */
+export type PropValue =
```
Comment (wavebeem) [inline src/api.ts:63]: I just realized this comment needs to be updated. I'll fix it tonight, then do a release.

**Comment by wavebeem** [comment]
@Vadimkin 3.1.0 is out! Thanks for your work on this :D https://www.npmjs.com/package/jsx-info

**Comment by Vadimkin** [comment]
@wavebeem thanks and congrats with one more release! 😀

**Pull request merged**